pkg/segment/segment.go

```diff
@@ -120,6 +120,18 @@ func (c *Client) Upload(data TelemetryData) error {
 		properties = properties.Set("error", data.Properties.Error).Set("error-type", data.Properties.ErrorType)
 	}
 
+	// send the Identify message data that helps identify the user on segment
+	err := c.SegmentClient.Enqueue(analytics.Identify{
+		UserId: strings.TrimSpace(string(userId)),
```

Comment:
We don't need `strings.TrimSpace(string(userId))`, it is already trimmed when returned by `getUserIdentity`.

Reply:
You can also remove `client *Client` arg from `getUserIdentity`.

Reply:
thx, updated